tools/schemacode/bidsschematools/render.py

```diff
@@ -205,11 +208,16 @@ def make_filename_template(schema, n_dupes_to_combine=6, **kwargs):
 
     paragraph = ""
     # Parent directories
-    paragraph += "{}-<{}>/\n\t[{}-<{}>/]\n".format(
-        schema["objects"]["entities"]["subject"]["name"],
-        schema["objects"]["entities"]["subject"]["format"],
-        schema["objects"]["entities"]["session"]["name"],
-        schema["objects"]["entities"]["session"]["format"],
+    paragraph += (
+        '<a href="SPEC_ROOT/99-appendices/09-entities.md#subject">'
```

Comment:
Don't you just want things like:

```suggestion
        '<a href="/99-appendices/09-entities.html#subject">'
```

Reply:
That works, thanks! I assumed we _didn't_ want to reference HTML paths or use absolute paths, but if this isn't the preferred approach we can always revisit it later.